child_compassion/migrations/10.0.1.3.0/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Théo Nikles <theo.nikles@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    # Update pictures older pictures that don't have a link
+    cr.execute("""
+        UPDATE compassion_child_pictures AS pics
+        SET image_url = CONCAT(
+            'https://erp.compassion.ch/web/image/compassion.child.pictures/',
+            id,
+            '/fullshot/child.jpg'
```

Comment:
I think I would prefer if we can name the file based on the date of the picture and the local id of the child. It would be like:
```
'/fullshot/', pics.date, '_', child.local_id, '.jpg'
```
It requires a join on the compassion_child table although...

Reply:
Fortunately, the table directly provides the child_id, so no join is required in that case !

Thank you for all the feedbacks